tests/unit/core/reducers/test_addons.js

```diff
@@ -105,4 +112,18 @@ describe('addon reducer', () => {
       baz: { description: null },
     });
   });
+
+  it('exposes "isRestartRequired" attribute from current version files', () => {
+    const addon = createFakeAddon({ extraFileProps: { is_restart_required: true } });
```

Comment:
Since this hits a special branch of the reducer code, I think it would be good to write a test for the opposite of this too, set up like:

```js
const addon = createFakeAddon({ extraFileProps: { is_restart_required: false } });
```